mmv1/products/monitoring/api.yaml

```diff
@@ -1757,6 +1757,25 @@ objects:
             - :NOT_CONTAINS_STRING
             - :MATCHES_REGEX
             - :NOT_MATCHES_REGEX
+            - :MATCHES_JSON_PATH
+            - :NOT_MATCHES_JSON_PATH
+        - !ruby/object:Api::Type::NestedObject
+          name: jsonPathMatcher
+          description: Information needed to perform a JSONPath content match.
+            Used for `ContentMatcherOption::MATCHES_JSON_PATH` and
+            `ContentMatcherOption::NOT_MATCHES_JSON_PATH`.
+          properties:
+          - !ruby/object:Api::Type::String
+            name: jsonPath
```

Comment:
This field should probably be required, right? If the user specifies json_path_matcher then they must also specify json_path?

Reply:
Yes that makes sense